oclubs/static-dev/css/content.pcss

```diff
@@ -0,0 +1,281 @@
+    font-size: 18px;
+  }
+}
+/* Post things: Jumbotron and well divs */
+/* Image gallery */
+.grid-item,
+.grid-sizer {
+  width: 23.5%;
+  margin-bottom: 2%;
+}
+
+.gutter-sizer {
+  width: 2%;
+}
+@media (max-width: 768px) {
+  .grid-item,
+  .grid-sizer {
+    width: 32%;
+  }
+}
+@media (max-width: 425px) {
+  .grid-item,
+  .grid-sizer {
+    width: 100%;
+    margin-bottom: 4%;
+  }
+}
+
+.photo_gallery_thumbnail {
+  margin-bottom: 0;
+}
+
+.loader-ellips {
+  font-size: 20px;
+  /* change size here */
+  position: relative;
+  width: 4em;
+  height: 1em;
+  margin: 10px auto;
+}
+
+.loader-ellips__dot {
+  display: block;
+  width: 1em;
+  height: 1em;
+  border-radius: 0.5em;
+  position: absolute;
+  animation-duration: 0.5s;
+  animation-timing-function: ease;
+  animation-iteration-count: infinite;
+}
+
+.loader-ellips__dot:nth-child(1),
+.loader-ellips__dot:nth-child(2) {
+  left: 0;
+}
+
+.loader-ellips__dot:nth-child(3) {
+  left: 1.5em;
+}
+
+.loader-ellips__dot:nth-child(4) {
+  left: 3em;
+}
+@keyframes reveal {
+  from {
+    transform: scale(0.001);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+@keyframes slide {
+  to {
+    transform: translateX(1.5em);
+  }
+}
+
+.loader-ellips__dot:nth-child(1) {
+  animation-name: reveal;
+}
+
+.loader-ellips__dot:nth-child(2),
+.loader-ellips__dot:nth-child(3) {
+  animation-name: slide;
+}
+
+.loader-ellips__dot:nth-child(4) {
+  animation-name: reveal;
+  animation-direction: reverse;
+}
+/* Image gallery */
+/* Club list refresh btn */
+#refresh_btn {
+  padding: 6px;
+  font-size: 12px;
+  color: #fff;
+  background-color: #5bc0de;
+  border-color: #46b8da;
+}
+@media (max-width: 768px) {
+  #refresh_btn {
+    font-size: 14px;
+  }
+}
+/* Club list refresh btn */
+/* Modal forms */
+.modal .form-group {
+  margin-bottom: 0;
+}
+/* Modal forms */
+/* Buttons */
+.btn_content {
+  margin: 10px;
+}
+
+/* .btn-group > .btn:first-child:not(:last-child):not(.dropdown-toggle) {
+/*  border-radius: 6px;
+/* }
+/* Buttons */
+/* loading gif */
+@keyframes lds-magnify {
+  0% {
+    -webkit-transform: translate(2px, 2px);
+    transform: translate(2px, 2px);
+  }
+
+  33.33% {
+    -webkit-transform: translate(102px, 2px);
+    transform: translate(102px, 2px);
+  }
+
+  66.66% {
+    -webkit-transform: translate(42px, 102px);
+    transform: translate(42px, 102px);
+  }
+
+  100% {
+    -webkit-transform: translate(2px, 2px);
+    transform: translate(2px, 2px);
+  }
+}
+@-webkit-keyframes lds-magnify {
+  0% {
+    -webkit-transform: translate(2px, 2px);
+    transform: translate(2px, 2px);
+  }
+
+  33.33% {
+    -webkit-transform: translate(102px, 2px);
+    transform: translate(102px, 2px);
+  }
+
+  66.66% {
+    -webkit-transform: translate(42px, 102px);
+    transform: translate(42px, 102px);
+  }
+
+  100% {
+    -webkit-transform: translate(2px, 2px);
+    transform: translate(2px, 2px);
+  }
+}
+
+.lds-magnify {
+  position: relative;
+}
+
+.lds-magnify > div {
```

Comment:
& > div

Reply:
This section of css is generated from a loading animation generation site.

Reply:
Then convert it